docker/files/webapp.conf

```diff
@@ -0,0 +1,8 @@
+server {
+  listen 80;
+  server_name wagn;
```

Comment:
I figured this out too.  You can pass --env MY_ENV_VAR=my_value in the docker run commands.  Then you just use the server's way of referencing the environment.  So wagh should be replaced by ${SERVER_NAME} (or whatever is correct syntax for nginx config files).